health_monitor/test/test_plugin_manager.cpp

```diff
@@ -52,7 +52,7 @@ namespace health_monitor
         //EXPECT_EQ(0, res.plugins.begin()->type);
         //EXPECT_EQ(0, res.plugins.begin()->versionId.compare(""));
         // for lane change plugin
-        EXPECT_EQ(false, std::prev(res.plugins.end())->activated);
+        EXPECT_EQ(1, std::prev(res.plugins.end())->activated);
```

Comment:
Why did this test case change from false to true? Your changes should not be side affecting plugin_manager behavior.

Reply:
I don't really know. When I ran the test cases in VSCode, I noticed that one failed for some reason.